milestone-2-demo.md

```diff
@@ -0,0 +1,80 @@
+# Milestone 2 demo
+
+It is strongly recommended to use Nix to run this demo and following description will use Nix setup.
+
+Main entry point of the demo is [Seath.Test.Main module](./test/Main.purs).
+
+## Running on local private network
+
+Local network capabilities are integrated into current setup. Local network provided by [Plutip](https://github.com/mlabs-haskell/plutip). Network is started from scratch on each run. Slot length is is to 1 second - see [config :: PlutipConfig](test/PlutipRunner.purs).
+
+To run the demo with local network do the following.
+
+From the root of the repository start Nix development shell:
+
+```shell
+nix develop --extra-experimental-features 'flakes nix-command' .#off-chain
+```
+
+Note, that features `flakes` and `nix-command` are required.
+
+When Nix shell is ready, switch to `off-chain` directory and start the execution:
+
+```shell
+cd off-chain
+spago run -m Seath.Test.Main -b plutip
+```
+
+Local network will be started automatically and as soon as all required wallets will be funded, execution of Seath demo will begin. Execution trace will be printed to the terminal.
+
+## Running on preprod testnet
+
+For preprod testnet [cardano transaction library runtime](https://github.com/Plutonomicon/cardano-transaction-lib/blob/v5.0.0/doc/runtime.md) is required. It includes Cardano node, Ogmios and Kupo.
+
+Demo setup has Nix run command to launch everything. Only requirement is `Docker` installed and running.
```

Comment:
is this true? I remember I had to add myself to some group for docker to work, maybe pointing to CTL documentation about this?

Reply:
Links to Docker setup instructions added.